### ISSUE TITLE
期待让RouterOS支持golang

### ISSUE THREAD

**Issue by amilus668** [elseif/MikroTikPatch#12]
感谢作者无私奉献🙏

**Comment by naishi001** [comment]
golang跨平台的

**Comment by elseif** [comment]
shell下可以运行go编译的程序。

**Comment by amilus668** [comment]
哇👍那我就可以试试部署 opengfw(¯▽¯)👍

**Comment by amilus668** [comment]
测试了一下，部署[OpenGFW](https://github.com/apernet/OpenGFW)

报错
```
/flash/rw/disk # ./OpenGFW-linux-amd64 -c config.yaml rules.yaml 
2024-06-25T10:58:45Z    FATAL   failed to parse config  {"error": "invalid config: io: exec: \"iptables\": executable file not found in $PATH"}
```

看起来是RouterOS没有iptables程序

**Comment by amilus668** [comment]
哈哈，看来是没救了🤦‍♂️

**Comment by Chupaka** [comment]
There are nftables modules in RouterOS kernel, so you just need `nft` command to try again: https://github.com/apernet/OpenGFW/issues/101#issuecomment-2016750202

**Comment by amilus668** [comment]
> There are nftables modules in RouterOS kernel, so you just need `nft` command to try again: [apernet/OpenGFW#101 (comment)](https://github.com/apernet/OpenGFW/issues/101#issuecomment-2016750202)

```
/flash/rw/disk # nft
bash: nft: not found
```

**Comment by Chupaka** [comment]
Yeah, you need to get it somewhere else. I just know that nftables work in Containers, so it does have support in kernel.

**Comment by amilus668** [comment]
> Yeah, you need to get it somewhere else. I just know that nftables work in Containers, so it does have support in kernel.

Are you running opengfw on Containers?

**Comment by pop2006** [comment]
能自由升级吗？如从7.15.1升级到7.15.2

**Comment by Chupaka** [comment]
> Are you running opengfw on Containers?

No, I was playing with WireGuard in Containers, it needs iptables, and I had to use iptables-legacy (in alpine linux) which is iptable interface for nftables
